tds_test.go

```diff
@@ -520,3 +521,37 @@ func TestBadConnect(t *testing.T) {
 		t.Error("Ping should fail for connection: ", badDSN)
 	}
 }
+
+func TestSSPIAuth(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("Only on windows")
+	}
+	checkConnStr(t)
+	connStr := makeConnStr(t)
+	params := connStr.Query()
+	params.Set("Integrated Security", "sspi")
+	connStr.RawQuery = params.Encode()
+
+	conn, err := sql.Open("mssql", connStr.String())
+	if err != nil {
+		t.Error("Open failed", err)
+	}
+	defer conn.Close()
+
+	stmt, err := conn.Prepare("select 1, 'abc'")
```

Comment:
Prepare offers very little benefit in SQL Server. Don't use it. 
Please use `db.QueryRow` directly.